src/pages/download.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import useDocusaurusContext from "@docusaurus/useDocusaurusContext";
+import Layout from "@theme/Layout";
+import Download from "@site/src/components/Download";
+
+export default function Home() {
+    const { siteConfig } = useDocusaurusContext();
+    return (
+        <Layout
+            title={siteConfig.title}
+            description="Download page for conda-forge infrastructure"
+        >
+            <main>
+                <h1 style={{
+                    marginTop: 10, textAlign: "center"
+                }}>
+                    Download conda-forge installer
```

Comment:
```suggestion
                    Download Miniforge
```

Reply:
There will probably be a lot of bikeshedding about the formulation on this page (what more explanatory text to include). I suggest we get the feature in, and let the core devs make such edits after.

Reply:
Although, 2c: "Conda-forge installer" is a lot easier to understand to a newcomer on this page than "Miniforge".